form-sharing/2-getSheetsList.js

```diff
@@ -0,0 +1,14 @@
+getValues(
+  '1s7K3kxzm5AlpwiALattyc7D9_aIyqWmo2ubcQIUlqlY',
+  'sheetsList of kobo forms!A:K'
+);
+fn(state => {
+  const { koboForms} = state;
+  const googleSheetsData = state.data.values.map(row => row[0]);
```

Comment:
I would call this `const sheetsUids` because you only care about first column.  Also the returned values contain array of arrays, then i would make sure to filter the first array out since that will be our headers. 

Something like this 👇🏽 

```
getValues(
  '1s7K3kxzm5AlpwiALattyc7D9_aIyqWmo2ubcQIUlqlY',
  'sheetsList of kobo forms!A:K',
  state => {
    const { koboForms, data } = state;
    const [headers, ...sheetsData] = data.values;
    const sheetsUids = sheetsData.map(row => row[0]);
    console.log('Ignoring headers', headers);
    console.log(sheetsUids);

    state.filteredKoboFormsData = koboForms.filter(
      form => !sheetsUids.includes(form.uid)
    );

    return state;
  }
);
```

Reply:
@hunterachieng just a heads up, `getValues()` does support callback. I have updated the [docs adaptors](https://github.com/OpenFn/adaptors/pull/505).